airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -330,8 +331,21 @@ def default_params(self) -> STEP_PARAMS:
         }
 
     def regression_tests_command(self) -> List[str]:
-        return [
-            "poetry",
+        """
+        This command
+        1. Starts a Google Cloud SQL proxy running on localhost, which is used by the connection-retriever to connect to postgres.
+        2. Gets the PID of the proxy so it can be killed once done.
+        3. Runs the regression tests.
+        4. Kills the proxy, and waits for it to exit.
+        5. Exits with the regression tests' exit code.
+
+        We need to explicitly kill the proxy in order to allow the GitHub Action to exit.
+
+        An alternative that we can consider is to run the proxy as a separate service.
```

Comment:
Nice, I've added that link. Since the existing code is working I'm going to leave it as-is for now but will circle back to try this out if time permits.